### PR TITLE
submission

### PR DESCRIPTION
**Pull request by harryshen225**
please review and advise. thx

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            name: "role",
+            message: "Role: ",
+            choices: [
+                "Manager",
+                "Engineer",
+                "Intern"
+            ]
+        },
+        {
+            when: response => (response.role === "Engineer") ? true : false,
```
Comment (mankinchi) [inline app.js]: I don't know 'bout this. Very nice, can you post this option on to #04 for people to explore this option? I believe this is so beautiful

File: app.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        ]);
+        createMoreUser = confirmRes.createMoreUser
+    }
+    return users;
+}
+
+function generateSingleCard(employee){
+    switch (employee.getRole()){
+        case "Manager":
+            return eval("`"+managerTemplate +"`");
```
Comment (mankinchi) [inline app.js]: You should never use `eval`. Like never ever. This is the evil of javascript. Try another method

File: app.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+function generateEmployeeCards(employeeArray){
+    const htmlEmployeeArray = employeeArray.map(employee => generateSingleCard(employee));
+    return `<div class="ui stackable cards">${htmlEmployeeArray.join(" ")}</div>`
+}
+
+function rankTeam(team){
+    const teamRanking = team.map(member =>{
+        switch(member.role){
+            case "Manager":
+                member.rank = 100;
```
Comment (mankinchi) [inline app.js]: why aren't this be a part of the Manager, Engineer, Intern class

**Comment by harryshen225** [comment]
opening new pull request

**Pull request closed without merging**